src/gui/addnewtorrentdialog.ui

```diff
@@ -364,33 +364,41 @@
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
-      <widget class="QProgressBar" name="progMetaLoading">
-       <property name="sizePolicy">
-        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
-         <horstretch>0</horstretch>
-         <verstretch>0</verstretch>
-        </sizepolicy>
-       </property>
-       <property name="maximum">
-        <number>0</number>
-       </property>
-       <property name="value">
-        <number>-1</number>
-       </property>
-       <property name="textVisible">
-        <bool>false</bool>
-       </property>
-      </widget>
-     </item>
-     <item>
-      <widget class="QLabel" name="lblMetaLoading">
-       <property name="sizePolicy">
-        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
-         <horstretch>0</horstretch>
-         <verstretch>0</verstretch>
-        </sizepolicy>
-       </property>
-      </widget>
+      <layout class="QHBoxLayout" name="metadataLayout">
+       <item>
+        <widget class="QProgressBar" name="progMetaLoading">
+         <property name="sizePolicy">
+          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
+           <horstretch>0</horstretch>
+           <verstretch>0</verstretch>
+          </sizepolicy>
+         </property>
+         <property name="maximum">
+          <number>0</number>
+         </property>
+         <property name="value">
+          <number>-1</number>
+         </property>
+         <property name="textVisible">
+          <bool>false</bool>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QLabel" name="lblMetaLoading">
+         <property name="text">
+          <string/>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QPushButton" name="buttonSave">
+         <property name="text">
+          <string>Save torrent file...</string>
```

Comment:
"torrent" -> ".torrent"?

Reply:
Are you sure we should use file extension to represent file type?

Reply:
I'm not sure either, you can wait for other opinions before further actions.

Reply:
IMO with "Save .torrent file" people understand that they are saving a bittorrent metadata file, even if they end up naming it with an extension other than .torrent.

If you really prefer, you could say instead "Save metadata file" or "Save Bittorrent metafile" or similar.

Reply:
Then maybe "Save to file" or even just "Save"?
If no I would prefer "Save .torrent file".

Reply:
> 
> 
> Then maybe "Save to file" or even just "Save"?
> If no I would prefer "Save .torrent file".

Since it's not originally a .torrent file, "Save **to** .torrent file" is more correct (I prefer .torrent instead of metadata).

Reply:
>"Save to .torrent file" 

"Save **as** .torrent file" ?

Reply:
> 
> 
> > "Save to .torrent file"
> 
> "Save **as** .torrent file" ?

Better (I need to get some sleep :wink: )